trojsten/submit/tests.py

```diff
@@ -639,3 +640,87 @@ def test_edit_submit_time(self):
         self.assertTrue(form.is_valid())
         edited_submit = form.save()
         self.assertEqual(edited_submit.time, new_time)
+
+
+class AllSubmitsListTets(TestCase):
+
+    def setUp(self):
+        self.url = reverse('all_submits_description_page')
+        self.grad_year = timezone.now().year + 1
+        self.non_staff_user = User.objects.create_user(username='jozko', first_name='Jozko',
+                                                       last_name='Mrkvicka', password='pass',
+                                                       graduation=self.grad_year)
+        self.staff_user = User.objects.create_user(username='staff', first_name='Staff',
```

Comment:
Nevyzera ze by sa niekde pouzivalo.

Celkovo toto vyzera ako odkopcene od niekial, bolo by pekne to trochu upratat a povyhadzovat co netreba.